examples/MvcWeb/TheraLang/Controllers/ProjectTypeController.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using MvcWeb.Services;
+using MvcWeb.TheraLang.Entities;
+
+namespace MvcWeb.TheraLang.Controllers
+{
+    [Route("api/ProjectType")]
+    [ApiController]
+    public class ProjectTypeController : ControllerBase
+    {
+        public ProjectTypeController(IProjectTypeService service)
+        {
+            _service = service;
+        }
+
+        private readonly IProjectTypeService _service;
+
+        [HttpPost]
+        [Route("create/{ProjectType}")]
+        public async Task<IActionResult> PostProjectType([FromBody]ProjectType projectType)
```

Comment:
we should not use entities her - but create separate models, but let it be for now